functions/src/api/v1/rooms/createRoom.ts

```diff
@@ -57,4 +58,4 @@ export const createRoom = async (req: Request, res: Response) => {
         roomId,
         roomSid,
     } as NewRoomResponse)
```

Comment:
i think it's not usefull to precise type

Reply:
I remember why, it's to clarify for the client part which interface/response to expect.

Reply:
ok

maybe would be better ?
```
async (req: Request, res: Response):Promise<NewRoomResponse> => {
...
 res.send({
        roomId,
        roomSid,
})
}
``

Reply:
I can't as I'm not returning but calling res.send function, so probably extend or type the Response if there is a geneneric

Reply:
You're right. I review on my phone :)